src/super_gradients/training/transforms/transforms.py

```diff
@@ -764,10 +770,27 @@ class DetectionTargetsFormatTransform(DetectionTransform):
         max_targets: int: max objects in single image, padding target to this size.
     """
 
+    _XYXY_LABEL = ConcatenatedTensorFormat(
```

Comment:
Whats this doing here ?
If I understand correctly, you are doing this so that you can pass ConcatenatedTensorFormat as a default? Why not using the defaults (by import, by extending resolve_param so that it'd take in account defaults...) plenty of solutions here.

Reply:
Yeah this was actually removed and in the latest commit the default value is used